### PR TITLE
fix bug around table borders and storybook tests

### PR DESCRIPTION
**Pull request by luthfib**
Fix bug around borders in table.

### CODE REVIEW

**Review by mrosata — DISMISSED** [review]

File: .changeset/wild-bananas-hammer.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@zus-health/ctw-component-library": patch
+---
+
+Fix bug around borders in table.
```
Comment (luthfib) [inline .changeset/wild-bananas-hammer.md:5]:
```
Fix bug around borders in table. Fix intermittent failing tests.
```

File: src/components/content/medications/patient-medications.stories.tsx [hunk 2]
```diff
@@ -86,14 +87,14 @@ export const TestCancelAddNewMed: StoryObj<Props> = {
     await medications.patientRecord.toHaveRowCount(1);
     const newMedication = "cabozantinib 20 MG Oral Capsule [Cometriq]";
     medications.clickAddMedication();
-    const addMedicationForm = medicationFormDrawer(canvasElement);
+    const addMedicationForm = await medicationFormDrawer(canvasElement);
```
Comment (mrosata) [inline src/components/content/medications/patient-medications.stories.tsx:90]: this isn't a promise

Comment (luthfib) [reply src/components/content/medications/patient-medications.stories.tsx:90]: It is now

Comment (mrosata) [reply src/components/content/medications/patient-medications.stories.tsx:90]: nvm you updated it

**Review by mrosata — APPROVED** [review]

**Pull request merged**